src/engine/interop/specification.md

```diff
@@ -0,0 +1,146 @@
+# Engine API. Interop edition
+
+This document specifies a subset of the Engine API methods that are required to be implemented for the Merge interop.
+
+*Note*: Sync API design is yet a draft and considered optional for the interop.
+
+## Underlying protocol
+
+This specification is based on [Ethereum JSON-RPC API](https://eth.wiki/json-rpc/API) and inherits the following properties of this standard:
+* Supported communication protocols (HTTP and WebSocket)
+* Message format and encoding notation
+* [Error codes improvement proposal](https://eth.wiki/json-rpc/json-rpc-error-codes-improvement-proposal)
+
+Client software **MUST** expose Engine API at a port independent from JSON-RPC API. The default port for the Engine API is 8550 for HTTP and 8551 for WebSocket.
+
```

Comment:
should the 8550/1 port _only_ serve engine endpoints or are we thinking it will offer a dedicated channel for the consensus engine to fetch from the necessary eth_* endpoints as well?

Reply:
Is your main concern that a client serving normal traffic on 8545 might become overloaded with traffic potentially causing consensus-critical requests to be dropped/delayed?